ansible/environments/local/group_vars/all

```diff
@@ -4,6 +4,8 @@
 openwhisk_tmp_dir: "{{ lookup('env', 'OPENWHISK_TMP_DIR')|default('/tmp', true) }}"
 config_root_dir: "{{ openwhisk_tmp_dir }}/wskconf"
 whisk_logs_dir: "{{ openwhisk_tmp_dir }}/wsklogs"
+coverage_enabled: "{{ lookup('env', 'GRADLE_COVERAGE') | default('false', true) | bool}}"
+coverage_logs_dir: "{{ openwhisk_tmp_dir }}/wskcov"
```

Comment:
Should these rather be set in the global group_vars/all to avoid having to copy these over into each and every environment?

Reply:
These are only used for Travis CI which uses local environment. Most likely we would not be computing coverage in other env as that would require more effort in copying back the coverage files.

So kept them in local to avoid polluting the global vars

Reply:
Is this not failing for other environments because these are missing? I see that coverage_enabled is checked for undefinedness, coverage_logs_dir however has no default in the template.

Reply:
All tasks which make use of `coverage_logs_dir` have a check for `coverage_enabled`. Further if `coverage_enabled is undefined` then `check if coverage collection is enabled` would set a default value to `false`. So this you not cause any issue in other env